backend/blueprints/spa_api/spa_api.py

```diff
@@ -28,7 +29,6 @@
 from .service_layers.player.player_profile_stats import PlayerProfileStats
 from .service_layers.player.player_ranks import PlayerRanks
 from .service_layers.queue_status import QueueStatus
-from .service_layers.replay.basic_stats import BasicStatChartData
```

Comment:
why was this file touched

Reply:
This is required, BasicStatChartData does not exist anymore.

Reply:
but why's PandasManager added as an import.

Reply:
o

Reply:
It was used in the old proto processing code (to verify we decoded correctly) but now its unused.